### PR TITLE
Use Blockstream API for Fetching Transactions

### PR DESCRIPTION
**Pull request by joelklabo**

### CODE REVIEW

File: code-ch11/tx.py [hunk 1]
```diff
@@ -22,14 +22,14 @@ class TxFetcher:
     @classmethod
     def get_url(cls, testnet=False):
         if testnet:
-            return 'http://testnet.programmingbitcoin.com'
+            return 'https://blockstream.info/testnet/api/'
```
Comment (waltertschwe) [inline code-ch11/tx.py:25]: you should move these into constants.

Comment (joelklabo) [reply code-ch11/tx.py:25]: Not a bad idea. I think to fix this I'd rather just make the minimum change here though.

Comment (waltertschwe) [reply code-ch11/tx.py:25]: :+1:

**Comment by joelklabo** [comment]
@jimmysong

**Review by jimmysong — APPROVED** [review]
Thanks guys! Good stuff.

**Comment by joelklabo** [comment]
Thanks @jimmysong just saw this got merged 👍

**Pull request merged**